packages/client/src/store/actions/organizationOperations.ts

```diff
@@ -13,17 +13,26 @@ import { enqueueNotification, showErrSnackbar } from "./appActions";
 
 const getOrganizationCollPath = () =>
   `${Collection.Organizations}/${__organization__}`;
-
+const getPublicOrganizationInfoCollPath = () =>
+  `${Collection.PublicOrgInfo}/${__organization__}`;
 export const updateOrganization =
   (
     orgData: OrganizationData,
     setSubmitting: (isSubmitting: boolean) => void
   ): FirestoreThunk =>
   async (dispatch) => {
     try {
+      const { displayName, location, emailFrom } = orgData;
       const db = getFirestore();
       const docRef = doc(db, getOrganizationCollPath());
+      const publicOrgInfoDocRef = doc(db, getPublicOrganizationInfoCollPath());
+
       await setDoc(docRef, orgData, { merge: true });
+      await setDoc(
+        publicOrgInfoDocRef,
+        { displayName, location, emailFrom },
+        { merge: true }
+      );
```

Comment:
Right...so I really wouldn't do it this way, but would rather store all of the data in organization document (`organizations/{orgName}`) and use a data trigger to (selectively) copy only the data that should be publicly available, over to `publicOrgInfo/{orgName}` (or which ever the path). 
The reason for this is that it simplifies the client's role in updating the organization data (and thus separates the concerns in somewhat of a right way) and enables us to reuse the data from org in both restricted mode (admin-only data) as well as public mode (available to non-registered users). An example of this would be a `from` email: used in admin-mode to send emails as well as in non-admin mode to display "Not registered, contact this email..." message.

To summarize, the steps I would take:
- [ ] Add all data to org info (organization document)
- [ ] Create a data trigger to selectively copy the data to publicly available doc 
- [ ] Reflect this in `firestore.rules`:
- anybody can read - `read: if true;`
- nobody can write - `write: if false; // or just the absence of write rule should do it` (data triggers and cloud functions aren't blocked by this rule)